docs/_docs/modifiers/modifiers.md

```diff
@@ -270,42 +270,60 @@ interface HelloWorld {
     fun MainFrame(
         @Design(node = "#button") button: @Composable (ComponentReplacementContext) -> Unit,
     )
+    @DesignComponent(node = "#MyButton")
+    fun MyButton()
 }
 ```
 
-The example below passes a `Composable` `fun` that accepts
-`software.vehicle.design.ComponentReplacementContext`. This object provides
-methods to access the style of the replaced element and its descendants.
-Component Replacements are provided with information on the component they are
-replacing including the modifiers that would be applied for the layout style,
-visual style, and (if applicable) the text style.
-
-Doing so allows a replacement component to take on the designer specified
+The `com.android.designcompose.ComponentReplacementContext` object provides
+a field to access the layout properties of the replaced element, and another field
+to access the text style if the replaced element is a text node. Doing so allows a
+replacement component to take on the designer specified size, position, and text
 appearance while offering more behavioral changes than are permitted with simple
 Modifier customizations. For example, replacing a styled text node with a
 complete text field.
 
 ```kotlin
 interface ComponentReplacementContext {
-    /// Return the layout modifier that would have been used to present this component.
-    /// This modifier doesn't contain any appearance information, and should be the first
-    /// in the modifier chain.
+    // Return the custom layout modifier that this component would have used so that the layout
+    // function can retrieve the component's layout properties. When replacing a node with a
+    // composable that is not a DesignCompose generated function, such as a simple Box() or an
+    // AndroidView, this modifier should be used as a modifier for that component in order for it to
+    // retain the original node's layout (size, position) properties.
     val layoutModifier: Modifier
 
-    /// Return the appearance modifier. This causes the view to be presented as specified
-    /// by the designer. It also includes any modifier customizations that may have been
-    /// specified elsewhere in the program.
-    val appearanceModifier: Modifier
-
-    /// Render the content of this replaced component, if any.
+    // Render the content of this replaced component, if any.
     @Composable fun Content(): Unit
```

Comment:
What does someone do with the `Content()` Composable? Where, how and why would they use it in their code?

Reply:
Ooh good point. I forgot about what `Content()` does. It actually gives the developer a way to compose the children of the original node, but it currently doesn't work. I forgot about this completely. @iamralpht what is the use case for this?

Since it doesn't work I'm wondering if we need to support it at this time. I did a quick test and I believe it doesn't work because of the way I fixed the component replacement crash by subscribing to layout after the fact. In doing so, if the replacement composable also called `Content()`, then `Content()` would subscribe to layout before its parent, and the Rust layout manager wouldn't be able to find the parent. It should probably error but right now it just think that it's a root node and adds it to a list of root nodes, but won't be used.

Reply:
Let's remove it. It used to be "I want to add interactivity to something by interposing an interactive view which takes the original children", but with Compose you can just use a Modifier for this use case (and we do still have the Modifier customization). So I don't think that we need this.

Reply:
done